services/drupal/web/modules/custom/epa_core/epa_core.module

```diff
@@ -373,13 +374,45 @@ function epa_core_tokens($type, $tokens, array $data, array $options, Bubbleable
           $term_description = $token->replace($term_description, ['term' => $data['term']]);
           $replacements[$original] = $term_description;
         }
-
         break;
     }
   }
   return $replacements;
 }
 
+function epa_core_addtocal_alter(array &$overridable_items) {
+  $overridable_items['override_status'] = TRUE;
+  $node = \Drupal::routeMatch()->getParameter('node');
+  if ($node->bundle() == 'event') {
+    $cal_description = $node->get('field_add_to_cal_description')->value;
+    $default_description = $node->get('field_description')->value;
+    if (!empty($cal_description)) {
+      $description = "$cal_description" . "\n";
+    } elseif (!empty($default_description)) {
+      $description = "$default_description" . "\n";
+    }
+    $event_type = Term::load($node->get('field_event_type')->target_id)->getName();
+    if (!empty($event_type)) {
+      $event_type_full = "Event Type: $event_type" . "\n";
+    }
+    $event_host = $node->get('field_event_host')->value;
+    if (!empty($event_host)) {
+      $event_host_full = "Event host: $event_host" . "\n";
+    }
+    $links = $node->get('field_links')->getValue();
+    $links_string = "Links:" . "\n";
+    foreach ($links as $link) {
+      $text = $link['title'];
+      $url = $link['uri'];
+      $links_string .= "$text" . ": " . $url . "\n";
```

Comment:
No need for the extra quotes around `"$text"`.  Also, if you're using double-quotes like this, no need for concatenation at all when referencing variables.  `$links_string .= "$text: $url\n";` will work just fine.  That said, explicit concatenation can assist with readability so that's fine